### PR TITLE
Remove "experiments" block, changes in variables, minor edits in examples

### PR DESCRIPTION
**Pull request by pablo19sc**
- Removed "experiments" block and update required_version to >= 1.3.0
- Updated variables to have "create_global_network" inside "global_network" block
- Minor updates in examples

### CODE REVIEW

**Comment by pablo19sc** [comment]
Passed tests:

test2
test1

File: variables.tf [hunk 1]
```diff
@@ -1,26 +1,45 @@
...
   }
 }
 
 # Core Network Variables
 variable "core_network" {
-  description = "Core Network information."
+  description = <<-EOF
+  Core Network definition. The following attributes are required:
+  - `description` = (string) Core Network's description.
+  - `description` = (any) Core Network's policy in JSON format. It is recommended the use of the [Core Network Document data source](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/networkmanager_core_network_policy_document)
```
Comment (drewmullen) [inline variables.tf:40]:
```suggestion
- `policy` = (any) Core Network's policy in JSON format. It is recommended the use of the [Core Network Document data source](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/networkmanager_core_network_policy_document)
```

**Review by drewmullen — CHANGES_REQUESTED** [review]
lg but 1 small change

**Review by drewmullen — APPROVED** [review]
looks like the example docs are a little fuzzy on which version is being used but i think thats because the v1 hasnt shipped yet

**Review by drewmullen — APPROVED** [review]

**Pull request merged**